playwright/_impl/_transport.py

```diff
@@ -35,9 +38,25 @@ def _get_stderr_fileno() -> Optional[int]:
 
 
 class Transport:
+    def __init__(self) -> None:
```

Comment:
Consider using abstract class using `ABC`

Reply:
Seems what I want, thanks for pointing that out!